ci-operator/step-registry/upi/install/metal/upi-install-metal-commands.sh

```diff
@@ -0,0 +1,261 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+export HOME=/tmp
+
+if [[ -z "$RELEASE_IMAGE_LATEST" ]]; then
+  echo "RELEASE_IMAGE_LATEST is an empty string, exiting"
+  exit 1
+fi
+
+# Ensure ignition assets are configured with the correct invoker to track CI jobs.
+export OPENSHIFT_INSTALL_INVOKER="openshift-internal-ci/${JOB_NAME_SAFE}/${BUILD_ID}"
+export TEST_PROVIDER='packet'
+export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="${RELEASE_IMAGE_LATEST}"
+
+# The oc binary is placed in the shared-tmp by the test container and we want to use
+# that oc for all actions.
+export PATH=${SHARED_DIR}:${PATH}
+
+cp "$(command -v openshift-install)" ${SHARED_DIR}
+mkdir ${SHARED_DIR}/installer
+
+if [[ "${CLUSTER_VARIANT}" =~ "mirror" ]]; then
+  export PATH=$PATH:${SHARED_DIR}  # gain access to oc
+  # mirror the release image and override the release image to point to the mirrored one
+  mkdir /tmp/.docker && cp /etc/openshift-installer/pull-secret /tmp/.docker/config.json
+  oc registry login
+  MIRROR_BASE=$( oc get is release -o 'jsonpath={.status.publicDockerImageRepository}' )
+  oc adm release new --from-release ${RELEASE_IMAGE_LATEST} --to-image ${MIRROR_BASE}-scratch:release --mirror ${MIRROR_BASE}-scratch || echo 'ignore: the release could not be reproduced from its inputs'
+  oc adm release mirror --from ${MIRROR_BASE}-scratch:release --to ${MIRROR_BASE} --to-release-image ${MIRROR_BASE}:mirrored
+  RELEASE_PAYLOAD_IMAGE_SHA=$(oc get istag ${MIRROR_BASE##*/}:mirrored -o=jsonpath="{.image.metadata.name}")
+  oc delete imagestream "$(basename "${MIRROR_BASE}-scratch")"
+  RELEASE_IMAGE_MIRROR="${MIRROR_BASE}@${RELEASE_PAYLOAD_IMAGE_SHA}"
+
+  echo "Installing from mirror override release ${RELEASE_IMAGE_MIRROR}"
+  OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="${RELEASE_IMAGE_MIRROR}"
+else
+  echo "Installing from release ${RELEASE_IMAGE_LATEST}"
+fi
```

Comment:
I assume we're not using a mirror variant? We can probably just remove this then.